cmake/deps.txt

```diff
@@ -17,12 +17,7 @@ coremltools;https://github.com/apple/coremltools/archive/refs/tags/7.1.zip;f1bab
 cxxopts;https://github.com/jarro2783/cxxopts/archive/3c73d91c0b04e2b59462f0a741be8c07024c1bc0.zip;6c6ca7f8480b26c8d00476e0e24b7184717fe4f0
 date;https://github.com/HowardHinnant/date/archive/refs/tags/v3.0.1.zip;2dac0c81dc54ebdd8f8d073a75c053b04b56e159
 dlpack;https://github.com/dmlc/dlpack/archive/5c210da409e7f1e51ddf445134a4376fdbd70d7d.zip;e499c86e4e5c5268a87661d7ea39c27fae10907c
-# This Eigen commit id matches the eigen archive being consumed from https://gitlab.com/libeigen/eigen/-/archive/3.4/eigen-3.4.zip
-# prior to the 3.4.1 RC changing the bits and invalidating the hash.
-# it contains changes on top of 3.4.0 which are required to fix build issues.
-# Until the 3.4.1 release this is the best option we have.
-# Issue link: https://gitlab.com/libeigen/eigen/-/issues/2744
-eigen;https://gitlab.com/libeigen/eigen/-/archive/1d8b82b0740839c0de7f1242a3585e3390ff5f33/eigen-1d8b82b0740839c0de7f1242a3585e3390ff5f33.zip;5ea4d05e62d7f954a46b3213f9b2535bdd866803
+eigen;https://github.com/eigen-mirror/eigen/archive/refs/tags/3.4.0.zip;d5bac4735f8d70065070834b298f222c194abd80
```

Comment:
Is the same commit available in GitHub? We want to make sure the code is same if possible